examples/IBM.Watson.Assistant.v2.Examples/IBM.Watson.Assistant.v2.Examples.csproj

```diff
@@ -6,10 +6,11 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="IBM.Cloud.SDK.Core" Version="0.8.0" />
+    <PackageReference Include="IBM.Cloud.SDK.Core" Version="1.0.0-rc1" />
   </ItemGroup>
 
   <ItemGroup>
+    <!-- <ProjectReference Include="..\..\..\dotnet-sdk-core\src\IBM.Cloud.SDK.Core\IBM.Cloud.SDK.Core.csproj" /> -->

```

Comment:
Just for verification, is this just commented out until the core changes merge to a public release?

Reply:
Yep, these will be here until the major release so I can reference the nuget package vs the local core reference!